src/components/Resource/Forms/Training/TrainingSubmit.tsx

```diff
@@ -419,9 +443,9 @@ export const TrainingFormSubmit = ({ model }: { model: NonNullable<TrainingModel
     },
   });
 
-  const handleSubmit = ({ ...rest }: z.infer<typeof schema>) => {
-    const userTrainingDashboardURL = `/user/${currentUser?.username}/models?section=training`;
+  const userTrainingDashboardURL = `/user/${currentUser?.username}/models?section=training`;
 
+  const handleSubmit = ({ ...rest }: z.infer<typeof schema>) => {
```

Comment:
Maybe you were trying to extract a field from here?
```suggestion
  const handleSubmit = (data: z.infer<typeof schema>) => {
```

Reply:
ah yeah, that was leftover from the old function